File: 11주차/KM/226-invert-binary-tree.java
class Solution {
    public TreeNode invertTree(TreeNode root) {
        if(root == null) return null;
        TreeNode temp = new TreeNode(root.val, root.left, root.right);
        temp.left = invertTree(root.right);
        temp.right = invertTree(root.left);
        return temp;
    }
}

File: 11주차/DE/226. Invert Binary Tree.js
function invert(node){
    if(node == null){
        return;
    }
    let temp = new TreeNode();

    temp = node.left;
    node.left = node.right;
    node.right = temp;
    invert(node.left);
    invert(node.right);
}

function invertTree(root) {
invert(root);
return root;
};